installers/rpm-code/codes.sh

```diff
@@ -102,6 +102,7 @@ codes_download() {
                 git clone $r -q "$repo"
                 cd "$d"
                 git checkout "$qualifier"
+                git submodule update --init --recursive
```

Comment:
What are the submodules. I see you are building quite a few other codes.

Reply:
https://github.com/openmc-dev/openmc/tree/2f068863d032d56152505c7d8ff6b969f4705003/vendor